docs/src/main/sphinx/connector/memsql.rst

```diff
@@ -115,6 +115,164 @@ that catalog name instead of ``singlestore`` in the above examples.
 Type mapping
 ------------
 
+Because Trino and Singlestore each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Singlestore to Trino read type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Singlestore types to the corresponding Trino types following
+this table:
+
+.. list-table:: Singlestore to Trino type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Singlestore database type
+    - Trino type
+    - Notes
+  * - ``BIT``
+    - ``BOOLEAN``
+    -
+  * - ``BOOLEAN``
+    - ``TINYINT``
+    -
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``INTEGER``
+    - ``INTEGER``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``DOUBLE PRECISION``
```

Comment:
Let's change to `DOUBLE`. `DOUBLE PRECISION` is the implementation detail, the type name doesn't exist in 
https://docs.singlestore.com/managed-service/en/reference/sql-reference/data-types/real-numbers.html